projects/gui/src/mainwindow.cpp

```diff
@@ -1034,6 +1047,43 @@ bool MainWindow::askToSave()
 	return true;
 }
 
+void MainWindow::adjudicateGame0()
+{
+	adjudicateGame(0);
+}
+
+void MainWindow::adjudicateGame1()
+{
+	adjudicateGame(1);
+}
+
+void MainWindow::adjudicateGame2()
+{
+	adjudicateGame(2);
+}
+
+void MainWindow::adjudicateGame(int value)
```

Comment:
Can you create an enum for the `value` parameter, eg. `MainWindow::UserAdjudication` etc?

Reply:
yes, cleaner code